video/out/opengl/video.c

```diff
@@ -3391,7 +3393,8 @@ static void check_gl_features(struct gl_video *p)
     bool have_mglsl = gl->glsl_version >= 130; // modern GLSL (1st class arrays etc.)
     bool have_texrg = gl->mpgl_caps & MPGL_CAP_TEX_RG;
     bool have_tex16 = !gl->es || (gl->mpgl_caps & MPGL_CAP_EXT16);
-    bool have_compute = gl->glsl_version >= 430; // easiest way to ensure all
+    bool have_compute = gl->DispatchCompute;
```

Comment:
The interesting part is that `GL_ARB_arrays_of_arrays` is a GLSL only extensions, there is no function to be loaded (no `gl->DispatchCompute`-like check). I also believe `GL_ARB_arrays_of_arrays` should be a separated capability, it's not a strict subset of compute shader I would say.